src/main/java/frc/robot/subsystems/ShooterHoodSubsystem.java

```diff
@@ -60,27 +62,35 @@ public ShooterHoodSubsystem() {
     hoodNEO.setIdleMode(IdleMode.kBrake);
     // The Hood needs to run in the negative direction *towards* the limit switch
     // If it runs the wrong way, flip this invert setting.
-    hoodNEO.setInverted(false);
+    hoodNEO.setInverted(true);
 
     reliableZero = false;
     stopHood();
+
+    CommandScheduler.getInstance().registerSubsystem(this);
   }
 
+  /*
+   * public void raiseShotAngle() { ++hoodSetpoint; }
+   * 
+   * public void lowerShotAngle() { --hoodSetpoint; }
+   */
+
   public void moveHood(double posRequest) {
     // Used for autonomous and vision-tied control of the shooter hood.
-    hoodPID.setReference(posRequest, ControlType.kSmartMotion, 0);
+    hoodPID.setReference(posRequest, ControlType.kPosition, 0);
```

Comment:
Why kPosition instead of kSmartMotion?

Reply:
kPosition is easier to get working correctly and is sufficient for a low-inertia system like the shooter hood.  It works just like a COTS servo in this mode.